scripts/plotting/tob_snp_chip_only_pca/plot_tob_snp_chip_pca_only.py

```diff
@@ -0,0 +1,90 @@
+"""Create PCA plots for the combined TOB-WGS/SNP-chip data"""
+
+from bokeh.io.export import get_screenshot_as_png
+from bokeh.resources import CDN
+from bokeh.embed import file_html
+from bokeh.transform import factor_cmap
+from bokeh.plotting import ColumnDataSource, figure
+import pandas as pd
+import hail as hl
+import click
+from analysis_runner import bucket_path, output_path
+
+SCORES = bucket_path('tob_snp_chip_pca/v0/scores.ht')
+EIGENVALUES = bucket_path('tob_snp_chip_pca/v0/eigenvalues.ht')
+TOB_WGS = bucket_path('mt/v3-raw.mt')
+
+
+@click.command()
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    scores = hl.read_table(SCORES)
+    tob_wgs = hl.read_matrix_table(TOB_WGS)
+    snp_chip_names = scores.s.collect()
+    wgs_names = tob_wgs.s.collect()
+
+    def sample_type(sample_name):
+        if sample_name in wgs_names:
+            prefix = 'dual_sample'
+        else:
+            prefix = 'snp_chip_only'
+
+        return prefix
+
+    labels = list(map(sample_type, snp_chip_names))
+
+    # get percent variance explained
+    eigenvalues = hl.import_table(EIGENVALUES)
+    eigenvalues = eigenvalues.to_pandas()
+    eigenvalues.columns = ['eigenvalue']
+    eigenvalues = pd.to_numeric(eigenvalues.eigenvalue)
+    variance = eigenvalues.divide(float(eigenvalues.sum())) * 100
+    variance = variance.round(2)
+
+    # Get number of PCs
+    number_of_pcs = len(eigenvalues)
+
+    # plot
+    cohort_sample_codes = list(set(labels))
+    tooltips = [('labels', '@label'), ('samples', '@samples')]
+    for i in range(0, (number_of_pcs - 1)):
+        pc1 = i
+        pc2 = i + 1
+        plot = figure(
+            title='Reprocessed Sample Projection',
+            x_axis_label='PC' + str(pc1 + 1) + ' (' + str(variance[pc1]) + '%)',
+            y_axis_label='PC' + str(pc2 + 1) + ' (' + str(variance[pc1]) + '%)',
```

Comment:
`pc2` instead of `pc1` for the y-axis?

```suggestion
            x_axis_label=f'PC{pc1 + 1} ({variance[pc1]})%)',
            y_axis_label=f'PC{pc2 + 1} ({variance[pc2]}%)',
```